### PR TITLE
Add admin user and home

### PR DESCRIPTION
**Pull request by roalcantara**
Add admin user flag
Add admin user password to secret file
Add admin user seed
Add admin routes
Add admin home controller
Configure admin users to be redirected to home controller after sign_in
Add sign in examples to application controller
Add admin base controller specs
Add sign in examples to admin base controller

### CODE REVIEW

File: features/support/paths.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+module NavigationHelpers
+  # Maps a name to a path. Used by the
+  #
+  #   When /^I go to (.+)$/ do |page_name|
+  #
+  # step definition in web_steps.rb
+  #
+  def path_to(page_name)
```
Comment (ddomingues) [inline features/support/paths.rb]: @roalcantara Are we using this helper?

**Comment by ddomingues** [comment]
@roalcantara What do you think if we merge this [commit](https://github.com/equalize-squad/equalize/pull/48/commits/ac8fa92ced31e207a9578aba2e9569cb5c9f1743) with the cucumber configuration commit?

File: features/support/paths.rb (hunk 1 above)
Comment (roalcantara) [inline features/support/paths.rb]: I don't think we are using it. I found it on the devise documentation, tho. It will be useful. Shall we get rid of it?

**Comment by roalcantara** [comment]
Hey @ddomingues, why do you think we should merge `Remove Coveralls integration` commit with the `Cucumber configuration`? Are they related, somehow?

File: app/controllers/admin/base_controller.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class Admin::BaseController < ApplicationController
+  before_action :authenticate_user!
+  before_action :require_admin
+
+  def require_admin
```
Comment (ddomingues) [inline app/controllers/admin/base_controller.rb]: I guess this method should be private. What do you reckon?

About syntax, take a look [here](https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier) about `single line`. 😉

File: Gemfile [hunk 3]
```diff
@@ -59,6 +59,7 @@ group :test do
   gem 'email_spec',             '~> 2.1'
   gem 'simplecov', require: false
   gem 'codecov', require: false
+  gem 'coveralls', require: false
```
Comment (ddomingues) [inline Gemfile]: @roalcantara I dunno why it appears here, but it is the argument to my previous comment 😉

File: spec/controllers/application_controller_spec.rb [hunk 4]
```diff
@@ -34,4 +35,34 @@ def index
       end
     end
   end
+
+  describe "user signs in" do
+    controller do
+      def after_sign_in_path_for(resource)
```
Comment (ddomingues) [inline spec/controllers/application_controller_spec.rb]: @roalcantara Why does the anonymous controller need to override the `ApplicationController#after_sign_in_path_for` method?

**Comment by ddomingues** [comment]
@roalcantara about syntax, what do you think if we follow this structure?

```
context "when it is a non admin user" do
      describe "after the sign in" do
        before do
          ...
        end
        
        it "should redirect to the root page" do
          ...
        end

        it "should redirect to the root page" do
          ...
        end
      end
    end
```

I'm talking about breakline 😬 

After that we can set it at rubocop :metal:

File: features/support/paths.rb (hunk 1 above)
Comment (ddomingues) [inline features/support/paths.rb]: Ok. If you think that will be useful let's keep it.

File: spec/controllers/application_controller_spec.rb (hunk 4 above)
Comment (roalcantara) [inline spec/controllers/application_controller_spec.rb]: It doesn't.  I gonna get rid of it!

**Comment by ddomingues** [comment]
@roalcantara about your question related to the merge, take a look [here](https://github.com/equalize-squad/equalize/pull/48/commits/a3b494b4ecadc97e4facf6b5b13bb43d664dfbce#r72734628).

**Pull request merged**